Packages/KeyAppKit/Sources/KeyAppNetworking/HTTPClient/HTTPResponseDecoder.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/// Decoder for response from `HTTPClient`
+public protocol HTTPResponseDecoder {
+    func decode<T: Decodable>(_ type: T.Type, data: Data, httpURLResponse response: HTTPURLResponse) throws -> T
+}
+
+/// ResponseDecoder for JSON type
+public class JSONResponseDecoder: HTTPResponseDecoder {
+    
+    private let jsonDecoder: JSONDecoder
+    
+    public init(jsonDecoder: JSONDecoder = .init()) {
```

Comment:
Add // MARK: - Init

Reply:
change `.init()` to `JSONDecoder()` because of micro slowing of compilation

Reply:
Done. Thanks!